tests/adapters/compliance_tests/add_bond.py

```diff
@@ -0,0 +1,28 @@
+from hamcrest import is_, assert_that
+from netman.core.objects.exceptions import BondAlreadyExist, BadBondNumber
+from tests.adapters.compliance_test_case import ComplianceTestCase
+
+
+class AddBondTest(ComplianceTestCase):
+    _dev_sample = "juniper"
+
+    def test_bond_is_created(self):
```

Comment:
"AddBond bond is created" is not a readable sentence.
"AddBond creates the bond" makes more sense
Therefore   test_creates_the_bond is what you need